frontend/src/components/pages/AccessControl/index.tsx

```diff
@@ -90,7 +90,13 @@ const AccessControlPage = (): JSX.Element => {
           .concat(" ", user.lastName.toLowerCase())
           .includes(search.toLowerCase()),
     );
-  }, [search, selectedFilter, users]);
+  }, [
+    search,
+    selectedFilter,
+    users,
+    UserStatusStates.ACTIVE_CAPITALIZED,
+    UserStatusStates.ALL,
```

Comment:
the other option is to just transform this into a callback but throwing these into the dep array should have no effect since the enums aren't being mutated